### PR TITLE
[Notifier] Added Sendinblue bridge documentation

### PR DESCRIPTION
**Pull request by drixs6o9**

### CODE REVIEW

File: notifier.rst [hunk 1]
```diff
@@ -66,6 +66,7 @@ OvhCloud    ``symfony/ovhcloud-notifier``     ``ovhcloud://APPLICATION_KEY:APPLI
 Sinch       ``symfony/sinch-notifier``        ``sinch://ACCOUNT_ID:AUTH_TOKEN@default?from=FROM``
 Smsapi      ``symfony/smsapi-notifier``       ``smsapi://TOKEN@default?from=FROM``
 Twilio      ``symfony/twilio-notifier``       ``twilio://SID:TOKEN@default?from=FROM``
+Sendinblue  ``symfony/sendinblue-notifier``   ``sendinblue://API_KEY@default?sender=PHONE``
```
Comment (OskarStark) [inline notifier.rst:69]: Please sort them alphabetically. Thanks

**Review by OskarStark — APPROVED** [review]

File: notifier.rst (hunk 1 above)
Comment (drixs6o9) [reply notifier.rst]: Done @OskarStark thanks for your review :)

**Comment by OskarStark** [comment]
Thank you Yann.

**Comment by ptondereau** [comment]
Does it fix #14288 ?

**Comment by OskarStark** [comment]
Yes thanks

**Pull request merged**